roles/preflight-checks/tasks/main.yml

```diff
@@ -0,0 +1,14 @@
+---
+- name: check ceph installed
+  command: ceph --version
+  register: result_ceph_installed
+  failed_when: false
+  delegate_to: "{{ groups['ceph_monitors'][0] }}"
+  when: ceph.enabled | default('False') | bool
```

Comment:
All tasks in ceph.yml depends on the result of this task, if we do this task in ceph.yml, we will have to add a extra condition for all tasks in ceph.yml. `when: result_ceph_installed.rc == 0`